test/e2e_node/runner/run_e2e.go

```diff
@@ -502,9 +502,11 @@ func getComputeClient() (*compute.Service, error) {
 }
 
 func deleteInstance(image string) {
-	_, err := computeService.Instances.Delete(*project, *zone, imageToInstanceName(image)).Do()
+	instanceName := imageToInstanceName(image)
```

Comment:
It is possible that instance deletion might fail due to GCE flake. Can we retry a couple of times just to be sure?